ote/src/cljs/ote/views/transit_visualization/change_utilities.cljs

```diff
@@ -35,9 +35,10 @@
     [:b "Taulukon ikonien selitteet"]]
    [:div (stylefy/use-style style/transit-changes-icon-legend-row-container)
     (doall
-      (for [[icon label] [[ote-icons/outline-add-box " Uusia vuoroja"]
-                          [ote-icons/outline-indeterminate-checkbox " Poistuvia vuoroja"]
-                          [ic/action-timeline " Pysäkkimuutoksia per vuoro"]
-                          [ic/action-query-builder " Aikataulumuutoksia per vuoro"]]]
+      (for [[icon color label] [[ic/content-remove-circle-outline {:color style/remove-color} "Mahdollisesti päättyvä reitti"]
```

Comment:
Old code I know, but still: a bit 90s to have marginal in localization string as a space character.:)
Alternatively could be done using styling or by prepending the space " " to the string here and not in the language string itself. The minor value there is it enables potential re-using of the string in some other view place i a need arises :)